REVIEW_GUIDELINE.md

```diff
@@ -0,0 +1,255 @@
+# Guideline for Pull Request Review
+
+## Introduction
+
+Code review is an essential part of the software development life cycle and is also one of the most challenging and time-consuming aspects of the software development process, requiring experienced team members to spend time reading, thinking about, assessing, and responding to new feature, fix, and system implementations.
+
+Here are some guidelines to assist you in reviewing pull requests.
+
+## Guidelines
+
+Before jumping straight to the code changes, some obvious things should be addressed first.
+
+**Pull Request Checklist:**
+
+1. Commit Message (spell checks, commit lint etc.)
+2. Appropriate title of pull request
+3. Number of commits (should be one commit, unless required)
+4. Check if there should be single pull request or multiple pull requests
+5. Source branch and target branch of pull request
```

Comment:
Can we please add a reference for the branch naming convention? You can also refer to this https://github.com/innovaccer/design-system/blob/master/CONTRIBUTING.md#branch-naming-conventions